src/components/resultswidget.cpp

```diff
@@ -0,0 +1,194 @@
+#include "resultswidget.h"
+
+ResultsWidget::ResultsWidget(QVector<cellResultsDisplayInfo> infos, QFrame *parent) : QFrame(parent)
+{
+
+    this->infos = infos;
+
+    // Total ResultsWidget Layout
+    resultsLayout = new QVBoxLayout;
+
+    // Layout for Buttons Only
+    resultsButtonLayout = new QGridLayout;
```

Comment:
Eventually I think we can make these buttons and their layout look a little better, if you have any ideas now go for it (I am no designer but they seem kinda big and spread out)

Reply:
I made it a bit better. They are all smaller and only take up the right half of the row.